ia/src/protocole.py

```
@@ -164,8 +180,52 @@ def execute_command(client, command, args) -> None:
         send_message(client, command)
     else:
         send_message(client, command)
-    if len(client.commands) < utils.MAX_COMMANDS:
-        client.commands.append(command)
+
+def initialize_clients(client) -> None:
```

Flake8 (7.2.0)

[error] expected 2 blank lines, found 1

(E302)